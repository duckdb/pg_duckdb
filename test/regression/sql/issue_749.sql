create temp table users (data jsonb);
insert into users values ('{"a": 123}');

create or replace function get_users() returns setof users as
$$ SELECT * FROM users; $$
language sql stable;

SET duckdb.force_execution = true;
-- This command used to crash due to DuckDB returning data in the json
-- format instead of in jsonb format.
SELECT * FROM ROWS FROM(get_users()) WITH ORDINALITY;
