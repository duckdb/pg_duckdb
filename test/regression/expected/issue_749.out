create temp table users (data jsonb);
insert into users values ('{"a": 123}');
create or replace function get_users() returns setof users as
$$ SELECT * FROM users; $$
language sql stable;
SET duckdb.force_execution = true;
-- This command used to crash due to DuckDB returning data in the json
-- format instead of in jsonb format.
SELECT * FROM ROWS FROM(get_users()) WITH ORDINALITY;
WARNING:  (PGDuckDB/CreatePlan) Prepared query returned an error: 'Not implemented Error: WITH ORDINALITY not implemented
    data    | ordinality 
------------+------------
 {"a": 123} |          1
(1 row)

